MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -71,8 +71,8 @@ open class BaseInstructionsBannerView: UIControl {
         delegate?.didTapInstructionsBanner(self)
     }
     
-    func set(_ instruction: VisualInstruction) {
-        let secondaryInstruction = instruction.secondaryTextComponents
+    func set(_ instruction: VisualInstruction?) {
```

Comment:
This optionality is needed in order to unset the instruction in `StepTableViewCell.prepareForReuse()`

/cc @frederoni

Reply:
also, should we use a different semantic here now that we've wrapped the primary and secondary components in the `VisualInstruction` class? Or does this feel idiomatic enough?

Reply:
Just `set` or `set(_:)` in this case almost feels like a reserved keyword but I don't have a strong opinion about that. Maybe `setInstruction(_:)` or `updateInstruction(_:)` would be more idiomatic?

Reply:
💯 % agree, @frederoni. `update(progress:)` or `update(instruction:)` would be pretty straightforward.

Reply:
Let's do this as tail work.